app/shared/components/Wallet/Status/Actions.js

```diff
@@ -0,0 +1,110 @@
+// @flow
+import React, { Component } from 'react';
+import { Header, Grid, Loader, Segment, Visibility } from 'semantic-ui-react';
+import { translate } from 'react-i18next';
+
+import ActionsTable from './Actions/Table';
+
+type Props = {
+  actions: {
+    getAccountActions: () => void
+  },
+  actionObjects: {},
+  t: () => void,
+  validate: {}
+};
+
+class Actions extends Component<Props> {
+  props: Props;
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      amount: 20
+    };
+  }
+
+  componentDidMount() {
+    this.tick();
+    this.interval = setInterval(this.tick.bind(this), 15000);
+  }
+
+  componentWillUnmount() {
+    clearInterval(this.interval);
+  }
+
+  loadMore = () => this.setState({ amount: this.state.amount + 20 }, () => {
```

Comment:
This should paginate, not just load more. The first request would be 20, the second would be 40 (duplicating the first 20), and so on.

Reply:
Yeah you are right, I updated the code to only pull 20 results at a time 😄